apps/extension/src/background/keyring/keyring.ts

```diff
@@ -575,6 +578,15 @@ export class KeyRing {
     return [];
   }
 
+  public async queryDefaultAccount(): Promise<DerivedAccount | undefined> {
+    const accounts = await this.queryAllAccounts();
```

Comment:
Should we use the method queryAccountById here?

Reply:
`queryAccountById`, despite its name, returns an array for some reason :/  I feel like it needs to be renamed